src/components/dropdownSearch/examples/DropdownSearchExamples.tsx

```diff
@@ -1,20 +1,53 @@
 import * as React from 'react';
 import * as _ from 'underscore';
+
+import {IReduxAction, ReduxConnect} from '../../../utils/ReduxUtils';
 import {UUID} from '../../../utils/UUID';
+import {IDropdownOption} from '../DropdownSearch';
+import {IOptionsDropdownSearchPayload, updateOptionsDropdownSearch} from '../DropdownSearchActions';
 import {DropdownSearchConnected} from '../DropdownSearchConnected';
 import {MultiSelectDropdownSearchConnected} from '../MultiSelectDropdownSearch/MultiSelectDropdownSearchConnected';
 
-export class DropdownSearchExamples extends React.Component<any, any> {
+interface DropdownSearchExamplesProps {
+    onOptionsChanged?: (id: string, options: IDropdownOption[]) => void;
+}
+
+const mapStateToProps = () => ({});
+
+const mapDispatchToProps = (dispatch: (action: IReduxAction<IOptionsDropdownSearchPayload>) => void): DropdownSearchExamplesProps => ({
+    onOptionsChanged: (id: string, options: IDropdownOption[]) => dispatch(updateOptionsDropdownSearch(id, options)),
+});
+
+@ReduxConnect(mapStateToProps, mapDispatchToProps)

```

Comment:
if you rewrite ReduxConnect as :  
  
```  
export class ReduxUtils {
    static mergeProps(...setOfProps: Array<KeyValue<any>>) {
        return extend({}, ...setOfProps);
    }
}  
```

I think you'll just be able to do this `@ReduxConnect(mapDispatchToProps)` without defining an empty mapStateToProps ,

Reply:
i think it would be a good idea for the future components we write as well, I know here it's just an example but overall it would be a good idea

Reply:
comment to make the comment up to date

Reply:
It is like that to be like the connect function which is:
```
connect<TStateProps, no_dispatch, TOwnProps>(
    mapStateToProps: MapStateToPropsParam<TStateProps, TOwnProps>,
    mapDispatchToProps: null | undefined,
    mergeProps: null | undefined,
    options: Options<TStateProps, TOwnProps>
)
```
`mapStateToProps` cannot be omitted.